src/vs/workbench/services/editor/common/editorGroupsService.ts

```diff
@@ -767,6 +773,34 @@ export interface IEditorGroup {
 	 */
 	isActive(editor: EditorInput | IUntypedEditorInput): boolean;
 
+	/**
+	 * Selects the editor in the group. If active is set to true,
+	 * it will be the active editor in the group.
+	 */
+	selectEditor(editor: EditorInput, active?: boolean): Promise<void>;
+
+
+	/**
+	 * Selects the editors in the group. If activeEditor is provided,
+	 * it will be the active editor in the group.
+	 */
+	selectEditors(editors: EditorInput[], activeEditor?: EditorInput): Promise<void>;
+
+	/**
+	 * Unselects the editor in the group. If the editor is not specified, unselects the active editor.
+	 */
+	unSelectEditor(editor: EditorInput): Promise<void>;
+
+	/**
+	 * Unselects the editors in the group. If the editor is not specified, unselects the active editor.
+	 */
+	unSelectEditors(editors: EditorInput[]): Promise<void>;
```

Comment:
Can we get this done without a `Promise<void>`?

Reply:
see above